encoding.bs

```diff
@@ -1038,6 +1051,33 @@ function decodeArrayOfStrings(buffer, encoding) {
 </div>
 
 
+<h3 id=interface-mixin-textdecoderattributes>Interface Mixin {{TextDecoderAttributes}}</h3>
```

Comment:
Nit: lowercase "mixin" in headings, here and below. (Headings are sentence case.)